python-scripts/xbox/interface/if_xbdm.py

```diff
@@ -77,21 +77,30 @@ def xbdm_command(cmd, length=None):
   return lines
 
 def GetModules():
-  return [] # FIXME: Make this work..
+  modulesList = {}
   lines = xbdm_command("modules")
```

Comment:
This already should return a list of strings, no need to do further parsing down below